examples/demo_workflow/README.md

```diff
@@ -18,16 +18,10 @@ It demonstrates the following APIs:
 
 You can install dapr SDK package using pip command:
 
-<!-- STEP
```

Comment:
This part would get executed prior to the workflow example and would override the local package (installed in tox.ini as pre-command), leading to new module not found